content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -201,7 +173,7 @@ This section assumes that you are using macOS X 10.0 or higher. ***TODO: okay?**
         mkdir -p /etc/opt/ripple ***TODO: changed this from ~/.config/ripple to match guidance coming in conf-file-location.md PR -- okay?***
         cp cfg/rippled-example.cfg /etc/opt/ripple/rippled.cfg
 
-{{n.next()}}. Edit `rippled.cfg` to set necessary file paths. The user you plan to run `rippled` as must have write permissions to all of the paths you specify here. ***TODO: I think we need to add this step to the centos and ubuntu tasks, correct? For centos and ubuntu, the config files are automatically placed in the correct directories, but the following paths still need to be manually updated to avoid permission issues, correct?***
+{{n.next()}}. Edit `rippled.cfg` to set necessary file paths. The user you plan to run `rippled` as must have write permissions to all of the paths you specify here. ***TODO: We need to add this step to the centos and ubuntu tasks, correct? For centos and ubuntu, the config files are automatically placed in the correct directories, but the following paths still need to be manually checked/updated to ensure that there are no permission issues, correct?***
```

Comment:
Yup, it's a good idea for the user to verify if they have the correct permissions rather than rippled failing to start and the user opening a support ticket for us.

Reply:
Maybe this can go as a tip to the user or debug steps if they cant start rippled